streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java

```diff
@@ -265,15 +264,6 @@
                               final Named named,
                               final Materialized<K, VOut, KeyValueStore<Bytes, byte[]>> materialized);
 
-    /**
-     * Create a new {@link SessionWindowedCogroupedKStream} instance that can be used to perform session
-     * windowed aggregations.
-     *
-     * @param sessionWindows the specification of the aggregation {@link SessionWindows}
-     * @return an instance of {@link SessionWindowedCogroupedKStream}
-     */
-    SessionWindowedCogroupedKStream<K, VOut> windowedBy(final SessionWindows sessionWindows);
-
```

Comment:
Just re-ordered this method to have the same ordering over multiple classes (time-window always before session-window)